sdk/identity/azure-identity/src/main/java/com/azure/identity/implementation/IdentityClientOptions.java

```diff
@@ -286,4 +287,14 @@ public IdentityClientOptions enablePersistentCache(boolean enabled) {
     public String getIntelliJKeePassDatabasePath() {
         return keePassDatabasePath;
     }
+
+
+    public IdentityClientOptions sharedTokenCacheCredential(boolean isSharedTokenCacheCred) {
```

Comment:
This setter probably better belongs in IdentityClientBuilder. After all, this is just an implementation detail instead of an "option" users can configure.

Reply:
It does require an extra parameter to IdentityClient's constructor that way.

Reply:
agreed and refactored.